### PR TITLE
Use BaseSchemaGenerator as a generator_class in get_schema_view

### PR DESCRIPTION
**Pull request by massover**
```python
# main.py
from rest_framework.schemas import get_schema_view
from rest_framework.schemas.coreapi import SchemaGenerator


class SchemaGenerator2(SchemaGenerator):
    pass

get_schema_view(
    generator_class=SchemaGenerator2
)
```

```bash
mypy  main.py

 Argument "generator_class" to "get_schema_view" has incompatible type "Type[SchemaGenerator2]"; expected "Type[SchemaGenerator]"
```

### CODE REVIEW

File: rest_framework-stubs/schemas/__init__.pyi [hunk 1]
```diff
@@ -18,7 +18,7 @@ def get_schema_view(
     renderer_classes: Optional[Sequence[Type[BaseRenderer]]] = ...,
     public: bool = ...,
     patterns: Optional[Sequence[Any]] = ...,
-    generator_class: Type[SchemaGenerator] = ...,
+    generator_class: Type[Union[openapi.SchemaGenerator, coreapi.SchemaGenerator]] = ...,
```
Comment (sobolevn) [inline rest_framework-stubs/schemas/__init__.pyi:21]: Maybe we can just go with `BaseSchemaGenerator`? https://github.com/encode/django-rest-framework/blob/0323d6f8955f987771269506ca5da461e2e7a248/rest_framework/schemas/generators.py#L146

**Review by sobolevn — COMMENTED** [review]
Thanks a lot!

File: rest_framework-stubs/schemas/__init__.pyi (hunk 1 above)
Comment (massover) [reply rest_framework-stubs/schemas/__init__.pyi]: 👍

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thanks!

**Pull request merged**